src/style.css

```diff
@@ -0,0 +1,41 @@
+:root {
+  --width-link: 16px;
+  --height-link: 16px;
+  --background-size: 13px;
+}
+
+body {
+  margin: 0 ;
+  padding: 0;
+}
+
+.nav {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__stars {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.stars__link {
+  width: var(--width-link);
+  height: var(--height-link);
+  background-image: url(images/star.svg);
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)